src/EnergyPlus/HeatRecovery.cc

```diff
@@ -3133,7 +3133,7 @@ namespace HeatRecovery {
             } break;
             case HXConfiguration::CrossFlowBothUnmixed: { // CROSS FLOW BOTH UNMIXED
                 Temp = Z * std::pow(NTU, -0.22);
-                Eps = 1.0 - std::exp((std::exp(-NTU * Temp) - 1.0) / Temp);
+                Eps = 1.0 - std::exp(std::expm1(-NTU * Temp) / Temp);
```

Comment:
It's mildly confusing to me that 

`expm1(n)` is the same as `exp(n) - 1`

but

`log1p(n)` is the same as `log(n - 1)`.

I understand the difference, but at first glance I thought maybe that was the cause of diffs here.

Reply:
These are supposed to be more precise. So consider these improvements that could possibly change the answer slightly.

    log1p(n) = log(1 + n)

I did these individually because there were diffs when I combined them all in 1 branch and I wanted to be sure which change caused which diffs.

Reply:
Indeed it all makes sense.  And it really can't be worth the time to investigate the diffs further.  You've done a great job already, I feel comfortable merging the three log/exp branches, as it is crystal clear that they are just precision improvements.